README.md

```diff
@@ -4,3 +4,6 @@ An example lesson teaching how to bake sourdough cinnamon rolls.
 
 Generated from [_crispy-doodle_](https://github.com/carpentries/crispy-doodle).
 
+## Contact
+
+If you have questions about the project, please open an issue or [contact the author](mailto:example@example.com).
```

Comment:
Thanks you for your suggestion, Toby. Good catch.  Just a slight suggested update to our preferred email address.

Reply:
```suggestion
If you have questions about the project, please open an issue or [contact the author](mailto:example2@example.com).
```